### PR TITLE
Elevator update

### PR DESCRIPTION
**Pull request by jhinspeter**

### CODE REVIEW

File: PowerUpRobot/src/org/usfirst/frc/team138/robot/Constants.java [hunk 1]
```diff
@@ -53,6 +53,11 @@
 		public static int LeftDriveEncoderPolarity = -1;
 		public static int RightDriveEncoderPolarity = 1;
 		
+		// Elevator
+		public final static boolean elevatorLimitSwitchActiveEngaged = false;
```
Comment (Team138Entropy) [inline PowerUpRobot/src/org/usfirst/frc/team138/robot/Constants.java]: Remove

File: PowerUpRobot/src/org/usfirst/frc/team138/robot/RobotMap.java [hunk 2]
```diff
@@ -16,6 +16,9 @@
 	public final static int RIGHT_CUBE_CAN_GRASPER_PORT = 4;
   
 	public final static int ELEVATOR_PORT = 2;
+	public final static int WINCH_PORT = 1;
+	public final static int RIGHT_INTAKE_PORT = 3;
```
Comment (Team138Entropy) [inline PowerUpRobot/src/org/usfirst/frc/team138/robot/RobotMap.java]: Remove

File: PowerUpRobot/src/org/usfirst/frc/team138/robot/RobotMap.java [hunk 3]
```diff
@@ -16,6 +16,9 @@
...
+	public final static int LEFT_INTAKE_PORT = 4;
```
Comment (Team138Entropy) [inline PowerUpRobot/src/org/usfirst/frc/team138/robot/RobotMap.java]: Remove

File: PowerUpRobot/src/org/usfirst/frc/team138/robot/Constants.java (hunk 1 above)
Comment (jhinspeter) [reply PowerUpRobot/src/org/usfirst/frc/team138/robot/Constants.java]: Done

File: PowerUpRobot/src/org/usfirst/frc/team138/robot/RobotMap.java (hunk 2 above)
Comment (jhinspeter) [reply PowerUpRobot/src/org/usfirst/frc/team138/robot/RobotMap.java]: Done

File: PowerUpRobot/src/org/usfirst/frc/team138/robot/RobotMap.java (hunk 3 above)
Comment (jhinspeter) [reply PowerUpRobot/src/org/usfirst/frc/team138/robot/RobotMap.java]: Done

**Pull request merged**